pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -157140,6 +157140,7 @@ self: {
          QuickCheck resourcet retry smallcheck store template-haskell
          temporary text th-reify-many transformers vector
        ];
+       enableSharedExecutables = false;
```

Comment:
this is the only actual change, the rest is just automatic trailing whitespace removal (feel free to disregard it of course)